python/foglamp/common/configuration_manager.py

```diff
@@ -426,6 +426,71 @@ def get_entry_val(k):
             err_response = ex.error
             raise ValueError(err_response)
 
+    async def update_configuration_item_bulk(self, category_name, config_item_list):
+        """ Bulk update config items
+
+        Args:
+            category_name: category name
+            config_item_list: dict containing config item values
+
+        Returns:
+            None
+        """
+
+        try:
+            payload = {"updates": []}
+            audit_details = {'category': category_name, 'items': {}}
+            for item_name, new_val in config_item_list.items():
+                if not isinstance(new_val, str):
+                    raise TypeError('new value should be of string type')
+
+                cat_item = await self._read_item_val(category_name, item_name)
```

Comment:
are we querying database for each item? We should read once for given category. and then use that for iteration

Reply:
Done